ports/qt5-base/vcpkg.json

```
@@ -39,6 +39,12 @@
       "dependencies": [
         "libmysql"
       ]
+    },
+    "vulkan": {
+      "description": "Build QtGui with vulkan",
```

```suggestion
      "description": "Enable Vulkan support in QtGui",
```

Does not build/link against anything Vlukan related. It might use headers but will not require any symbols.